src/classes/GE_PaymentServices.cls

```diff
@@ -100,6 +100,10 @@ public with sharing class GE_PaymentServices {
         return configService.getGatewayIds();
     }
 
+    public String getMerchantIds() {
+        return configService.getMerchantIds();
+    }
+
```

Comment:
Missing another method:
```
/**
     * @description Returns JSON Web Token to payments services tokenize card iframe
     * @return String
     */
    public String getJwtForToken() {
        return new PS_Request.Builder().getJWT(PS_Request.ElevateEndpoint.TOKENIZE);  
    }
    
```

Reply:
@RadSFDeveloper yeah thanks. Was holding off until I made `getJWT()` public in `PS_Request`. Just updated!